<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>addwindow</class>
    <widget class="QMainWindow" name="addwindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>450</width>
                <height>520</height>
            </rect>
        </property>
        <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>45</horstretch>
                <verstretch>0</verstretch>
            </sizepolicy>
        </property>
        <property name="minimumSize">
            <size>
                <width>450</width>
                <height>520</height>
            </size>
        </property>
        <property name="maximumSize">
            <size>
                <width>450</width>
                <height>520</height>
            </size>
        </property>
        <property name="windowTitle">
            <string>addwindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <widget class="QGroupBox" name="buttBox">
                <property name="geometry">
                    <rect>
                        <x>5</x>
                        <y>360</y>
                        <width>440</width>
                        <height>146</height>
                    </rect>
                </property>
                <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                    </sizepolicy>
                </property>
                <property name="minimumSize">
                    <size>
                        <width>440</width>
                        <height>0</height>
                    </size>
                </property>
                <property name="maximumSize">
                    <size>
                        <width>440</width>
                        <height>16777215</height>
                    </size>
                </property>
                <property name="title">
                    <string/>
                </property>
                <widget class="QPushButton" name="saveButton">
                    <property name="geometry">
                        <rect>
                            <x>270</x>
                            <y>50</y>
                            <width>75</width>
                            <height>40</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>14</pointsize>
                        </font>
                    </property>
                    <property name="text">
                        <string>保存</string>
                    </property>
                </widget>
                <widget class="QPushButton" name="clearButton">
                    <property name="geometry">
                        <rect>
                            <x>100</x>
                            <y>50</y>
                            <width>75</width>
                            <height>40</height>
                        </rect>
                    </property>
                    <property name="font">
                        <font>
                            <pointsize>14</pointsize>
                        </font>
                    </property>
                    <property name="text">
                        <string>清空</string>
                    </property>
                </widget>
            </widget>
            <widget class="QGroupBox" name="inputBox">
                <property name="geometry">
                    <rect>
                        <x>5</x>
                        <y>5</y>
                        <width>440</width>
                        <height>350</height>
                    </rect>
                </property>
                <property name="minimumSize">
                    <size>
                        <width>440</width>
                        <height>350</height>
                    </size>
                </property>
                <property name="maximumSize">
                    <size>
                        <width>440</width>
                        <height>350</height>
                    </size>
                </property>
                <property name="title">
                    <string/>
                </property>
                <widget class="QWidget" name="layoutWidget">
                    <property name="geometry">
                        <rect>
                            <x>80</x>
                            <y>70</y>
                            <width>291</width>
                            <height>142</height>
                        </rect>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                        <property name="horizontalSpacing">
                            <number>30</number>
                        </property>
                        <property name="verticalSpacing">
                            <number>10</number>
                        </property>
                        <item row="0" column="0">
                            <widget class="QLabel" name="idLabel">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>14</pointsize>
                                        <bold>false</bold>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>ID</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="idLineEdit">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="ipLabel">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>14</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>地址</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QLineEdit" name="ipLineEdit">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="0">
                            <widget class="QLabel" name="portLabel">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="font">
                                    <font>
                                        <pointsize>14</pointsize>
                                    </font>
                                </property>
                                <property name="text">
                                    <string>端口</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QLineEdit" name="portLineEdit">
                                <property name="minimumSize">
                                    <size>
                                        <width>0</width>
                                        <height>40</height>
                                    </size>
                                </property>
                                <property name="maximumSize">
                                    <size>
                                        <width>16777215</width>
                                        <height>40</height>
                                    </size>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </widget>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
